packages/types/src/kilocode/kilocode.ts

```diff
@@ -8,6 +8,7 @@ export const ghostServiceSettingsSchema = z
 		enableQuickInlineTaskKeybinding: z.boolean().optional(),
 		enableSmartInlineTaskKeybinding: z.boolean().optional(),
 		showGutterAnimation: z.boolean().optional(),
+		onlyAdditions: z.boolean().default(true).optional(),
```

Comment:
this isnt actually a setting atm. Do you intend to enable people to disable it? If they do the ghost won't work anymore right? Aren't we now hiding deade code behind onlyAdditions = false?

Reply:
But I removed this for now, and disabled the call to displaySuggestions (SVG decorations) in the GhostProvider.